src/clj/rems/application/search.clj

```diff
@@ -29,9 +32,48 @@
 
 (defn- index-application! [^IndexWriter writer app]
   (log/info "Indexing application" (:application/id app))
-  (let [doc (Document.)]
-    (.add doc (StringField. "id" (str (:application/id app)) Field$Store/YES))
-    (.add doc (TextField. "applicant" (str (:application/applicant app)) Field$Store/NO))
+  (let [doc (Document.)
+        id (->> [(:application/id app)
```

Comment:
minor: how about making a separate `index-terms-for-application` function that returns a map like
```
{:id "3 2019/3"
 :applicant "Alice Applicant alice@appli.cant"
 ...}
```
This would be slightly cleaner, and also you could implement the `"all"` field by catenating all values of this map

Reply:
👍